images/macos/provision/utils/utils.sh

```diff
@@ -173,6 +173,38 @@ brew_smart_install() {
         brew install --build-from-source $tool_name
     else
         echo "Downloading $tool_name..."
+
+        # get deps & cache em
```

Comment:
This comment is not clear

Reply:
"comment is not clear" does not help to improve it.
feel free to suggest better wording